regression/goto-analyzer/value-set-addition-05/test.desc

```diff
@@ -0,0 +1,9 @@
+CORE
+main.c
+--variable-sensitivity --vsd-values set-of-constants --show
+^EXIT=0$
+^SIGNAL=0$
+main::1::p .* value-set-begin: 2, 3 :value-set-end
+main::1::q .* value-set-begin: 5, 10 :value-set-end
+main::1::t .* value-set-begin: 7, 8, 12, 13 :value-set-end
```

Comment:
This line is especially interesting as it shows that the abstraction is not relational; the link between `p` and `q` is not maintained after the merge at the end of the `if`, so you wind up with false alarms like `12`.